connector/src/main/java/org/geysermc/connector/entity/Entity.java

```diff
@@ -262,6 +262,11 @@ public void updateBedrockAttributes(GeyserSession session) {
         session.sendUpstreamPacket(updateAttributesPacket);
     }
 
+    /**
+     * Applies the Java metadata to the local Bedrock metadata copy
+     * @param entityMetadata the Java entity metadata
+     * @param session GeyserSession
+     */
     public void updateBedrockMetadata(EntityMetadata entityMetadata, GeyserSession session) {
```

Comment:
As we use this method elsewhere outside of the java metadata translator, have you made sure that this has no unintended side effects (such as with entity mounting)?

Reply:
Entity mounting is fine. In general this should be a non-issue as we don't use `updateBedrockMetadata(EntityMetadata entityMetadata, GeyserSession session)` outside of JavaEntityMetadataTranslator.